templates/workloads/services/lb-web/cf.yml

```diff
@@ -268,6 +268,31 @@ Resources:
       ServiceToken: !GetAtt RulePriorityFunction.Arn
       ListenerArn: !GetAtt EnvControllerAction.HTTPSListenerArn
 
+  HTTPListenerRuleWithDomain:
```

Comment:
We should also disable the path based HTTP rule at the bottom of this file.

Reply:
Its not enabled if using HTTPS https://github.com/aws/copilot-cli/pull/2153/files#diff-3e5a9c72c979131b5776cb4d5ff4f878e750776dcb64b5048fae79410b5cdbe8R324